README.md

```diff
@@ -133,18 +140,60 @@ In addition, you might need to install Corepack. For more information checkout [
    $ yarn start
    ```
 
+> Note: This command will also open up Storybook locally. This provides you with a full list of available components that you can experiment with. Each component in Storybook also contains detailed documentation how to use them and the parameters that they can accept.
+
 ### Setup
-Note: This is not filled out yet!
+
+> Note: To be able to edit tokens, you must own a pro license of token studio
+
+1. First, open your Copy of the B01LER and bookmark Tokens Studio in the Plugins panel
```

Comment:
The consistency with the presence of full-stop would be nice. Right now some of them has it and some of them lacks it.